### ISSUE TITLE
Design feedback - Design intent Collaboration COVID-19 Vaccine Trials Content and Tooling

### ISSUE THREAD

**Issue by CrystabelReiter** [department-of-veterans-affairs/va.gov-team#11992]
**Do:** Mandatory feedback
**Try:** Exploratory next step
**Consider:** Sharing ideas/alternate ways of thinking

**Do:**
- No "Do" feedback currently

**Try:**

- Adding the note sections with a blue bg alert style (https://design.va.gov/components/alertboxes)
- For the race and gender questions, the question seems to feel lost in the hierarchy with the Note section. Maybe add the notes as blue bg alerts in the suggestion above or change the type hierarchy of the questions. 

**Consider:**

- Consider for the contact information section, using the required fields instead of having (all fields required in parentheses) -- I saw in the staging prototype in the meeting that some required fields were showing. 

**Definition of Done:**
- [x] Incorporate the **Do** feedback, and let us know on this ticket when they are updated mockups to check out 

- [x] Leave any comments for feedback you decide _not_ to take

Emily or Crystabel will close the ticket once those things are all set. Thank you!

**Comment by artsymartha68** [comment]
Adding @stephenBarrs to implement the required fields feedback above. Again whatever the default behavior is, we'll go with. It originally looked like overkill to my eyes to have ALL fields marked with a red "Required" but I am willing to go with the flow here.   👍 

Agree with the Race/Gender question info, again I think these form components have the correct styling by default vs. my UX Pin prototype. Will verify with @stephenBarrs when it comes to life in the code!

**Comment by CrystabelReiter** [comment]
@artsymartha68 Thanks Martha! I agree that having optional fields marked only seems more friendly. I did a required/optional field audit a while ago on whether we should do away with required fields and only mark optional. According to this [article](https://www.formulate.com.au/blog/required-versus-optional-fields-new-standard/), it's recommended that backgrounder forms have the required markers only, so we kept it to that pattern. We could bring this topic to the design system council though! 
Thanks for looping in @stephenBarrs and commenting. I'll close now
